Updated dependencies to latest.

leifnel opened the pull request:
Converted slave id from string to number, libraries require that now.

leifnel committed:
Updated dependencies to latest.
Converted slave id from string to number, libraries require that now.

leifnel commented:
Fixes issue #25

leifnel committed:
Print err.message instead of [object Object]

leifnel committed:
Print entire err object

hardillb commented:
@davidoc85 Can you guys test this please, I don't have any modbus kit anymore

leifnel commented:
Please commit this.

hardillb commented:
@davidoc85 @gedtyrrell Guys are you going to get round to looking at this?

hardillb commented:
@leifnel Since we have had no response here I propose we move this to your repo. If you have a build you are happy with bump the version number and add yourself as a contributor to the `package.json` as well as updating the git repo entry.

Once done PM me on Slack and I'll pull from your repo and push it to npm and add you as a maintainer so you can push further updates

leifnel committed:
Latest dependencies

leifnel committed:
Changed url to github.com/leifnel...

Alkarex committed:
Ensure setId is integer

`node.client.setID(...)` must be an integer. At the moment, the parameter comes from `obj.id`, which might be a string.
So this PR forces `obj.id` to be a number.

leifnel committed:
Merge pull request #1 from Alkarex/ensure-setID-integer

Ensure setId is integer
